src/components/Fab/index.ios.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {
+    KeyboardAvoidingView,
+} from 'react-native';
+
+import FAB from './FAB';
+import FabPropTypes from './FabPropTypes';
+
+function Fab({onPress, isActive}) {
+    return (
+        <KeyboardAvoidingView behavior="position">
```

Comment:
In [our documentation](https://github.com/Expensify/Expensify.cash#Philosophy), in the Cross Platform 99.999% section, we strongly discourage platform specific code. I understand that the problem is only happening on iOS but this does not mean our solution needs to be iOS specific. Does `KeyboardAvoidingView` only work on iOS? If not, then we don't need this ios specific file.

Reply:
@Luke9389 for android we have no need for `KeyboardAvoidingView ` and we need another configuration for `KeyboardAvoidingView` if we want to add so that's the reason I moved to platform-specific code.